bot/exts/moderation/silence.py

```diff
@@ -102,61 +128,112 @@ async def unsilence(self, ctx: Context) -> None:
 
         If the channel was silenced indefinitely, notifications for the channel will stop.
         """
-        await self._get_instance_vars_event.wait()
+        await self._init_task
         log.debug(f"Unsilencing channel #{ctx.channel} from {ctx.author}'s command.")
-        if not await self._unsilence(ctx.channel):
-            await ctx.send(f"{Emojis.cross_mark} current channel was not silenced.")
+        await self._unsilence_wrapper(ctx.channel)
+
+    async def _unsilence_wrapper(self, channel: TextChannel) -> None:
+        """Unsilence `channel` and send a success/failure message."""
+        if not await self._unsilence(channel):
+            overwrite = channel.overwrites_for(self._verified_role)
+            if overwrite.send_messages is False and overwrite.add_reactions is False:
```

Comment:
How would you feel about making this an `or`? To get the manual message out if either was disabled manually. I think when mods silence manually, they may not always disable reactions.

Reply:
Changed in bfab492